### ISSUE TITLE
"An error occurred loading the update" during installation

### ISSUE THREAD

**Issue by bigmmind** [BenSova/Patched-Sur#464]
Hi! I'm getting this error while trying to install Big Sur on my MacBook Pro Mid 2012 (currently on latest Catalina)
I tried re-creating the usb stick from scratch using the latest release of the patched, tried barrykn method by manually patching through Terminal, still the same. 
What do you suggest? Thanks a lot in advance.

[Error screenshot ](https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png)

**Comment by ozziepadilla** [comment]
> Hi! I'm getting this error while trying to install Big Sur on my MacBook Pro Mid 2012 (currently on latest Catalina)
> I tried re-creating the usb stick from scratch using the latest release of the patcher, tried barrykn method by manually creating USB and patching with micropatcher through Terminal, still the same.
> I click OK in the error box, restart the computer and Catalina loads normally. So it looks like installation doesn't even start.
> What do you suggest? Thanks a lot in advance.
> 
> [Error screenshot](https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png)

With patchet Sur you don’t need to create the usb manually so that Barry is another way but isn’t patchet Sur by benSova

**Comment by ozziepadilla** [comment]
> Hi! I'm getting this error while trying to install Big Sur on my MacBook Pro Mid 2012 (currently on latest Catalina)
> I tried re-creating the usb stick from scratch using the latest release of the patcher, tried barrykn method by manually creating USB and patching with micropatcher through Terminal, still the same.
> I click OK in the error box, restart the computer and Catalina loads normally. So it looks like installation doesn't even start.
> What do you suggest? Thanks a lot in advance.
> 
> [Error screenshot](https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png)

You have to use patchet Sur

**Comment by bigmmind** [comment]
> > Hi! I'm getting this error while trying to install Big Sur on my MacBook Pro Mid 2012 (currently on latest Catalina)
> > I tried re-creating the usb stick from scratch using the latest release of the patcher, tried barrykn method by manually creating USB and patching with micropatcher through Terminal, still the same.
> > I click OK in the error box, restart the computer and Catalina loads normally. So it looks like installation doesn't even start.
> > What do you suggest? Thanks a lot in advance.
> > [Error screenshot](https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png)
> 
> You have to use patched Sur

Yes, that's what I did and this error occurred. I am just saying that I tried another patch and got the same error. So same error with both patchers, including Patched Sur.

**Comment by ozziepadilla** [comment]
> > > Hi! I'm getting this error while trying to install Big Sur on my MacBook Pro Mid 2012 (currently on latest Catalina)
> > > I tried re-creating the usb stick from scratch using the latest release of the patcher, tried barrykn method by manually creating USB and patching with micropatcher through Terminal, still the same.
> > > I click OK in the error box, restart the computer and Catalina loads normally. So it looks like installation doesn't even start.
> > > What do you suggest? Thanks a lot in advance.
> > > [Error screenshot](https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png)
> > 
> > 
> > You have to use patched Sur
> 
> Yes, that's what I did and this error occurred. I am just saying that I tried another patch and got the same error. So same error with both patchers, including Patched Sur.

Did you use a usb or micro sdcard? Make sure the usb is right and 16 gigs

**Comment by bigmmind** [comment]
USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition

**Comment by ozziepadilla** [comment]
> USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition

Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it in holding down option key select EFI boot Mac will shut down then turn it on again  then select install macOS Big Sur and let it finish

**Comment by ozziepadilla** [comment]
> > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > 
> > 
> > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> 
> Thank you, that is exactly how I did it, but still getting the error

Can take a picture with ur phone to look the error

**Comment by bigmmind** [comment]
> > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > 
> > > 
> > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > 
> > 
> > Thank you, that is exactly how I did it, but still getting the error
> 
> Can take a picture with ur phone to look the error

Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png

**Comment by ozziepadilla** [comment]
> > > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > > 
> > > > 
> > > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > > 
> > > 
> > > Thank you, that is exactly how I did it, but still getting the error
> > 
> > 
> > Can take a picture with ur phone to look the error
> 
> Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png

Your Mac 💻 is connected to the wifi right now ? If yes or not try the process with the Ethernet cable connected directly from modem to your Mac and let me know how’s going

**Comment by ozziepadilla** [comment]
> > > > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > > > 
> > > > > 
> > > > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > > > 
> > > > 
> > > > Thank you, that is exactly how I did it, but still getting the error
> > > 
> > > 
> > > Can take a picture with ur phone to look the error
> > 
> > 
> > Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png
> 
> Your Mac 💻 is connected to the wifi right now ? If yes or not try the process with the Ethernet cable connected directly from modem to your Mac and let me know how’s going

Seems you have very slow internet service usually takes few minutes downloading macOS Big Sur the installation takes about 25 to 30 minutes and few minutes optimizing macOS Big Sur

**Comment by bigmmind** [comment]
> > > > > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > > > > 
> > > > > > 
> > > > > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > > > > 
> > > > > 
> > > > > Thank you, that is exactly how I did it, but still getting the error
> > > > 
> > > > 
> > > > Can take a picture with ur phone to look the error
> > > 
> > > 
> > > Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png
> > 
> > 
> > Your Mac 💻 is connected to the wifi right now ? If yes or not try the process with the Ethernet cable connected directly from modem to your Mac and let me know how’s going
> 
> Seems you have very slow internet service usually takes few minutes downloading macOS Big Sur the installation takes about 25 to 30 minutes and few minutes optimizing macOS Big Sur

Thanks a lot ! I will try with Ethernet connected and wifi turned off.

**Comment by bigmmind** [comment]
> > > > > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > > > > 
> > > > > > 
> > > > > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > > > > 
> > > > > 
> > > > > Thank you, that is exactly how I did it, but still getting the error
> > > > 
> > > > 
> > > > Can take a picture with ur phone to look the error
> > > 
> > > 
> > > Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png
> > 
> > 
> > Your Mac 💻 is connected to the wifi right now ? If yes or not try the process with the Ethernet cable connected directly from modem to your Mac and let me know how’s going
> 
> Seems you have very slow internet service usually takes few minutes downloading macOS Big Sur the installation takes about 25 to 30 minutes and few minutes optimizing macOS Big Sur

Just tried it, same error happens with Ethernet connection.

**Comment by ozziepadilla** [comment]
> > > > > > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > > > > > 
> > > > > > > 
> > > > > > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > > > > > 
> > > > > > 
> > > > > > Thank you, that is exactly how I did it, but still getting the error
> > > > > 
> > > > > 
> > > > > Can take a picture with ur phone to look the error
> > > > 
> > > > 
> > > > Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png
> > > 
> > > 
> > > Your Mac 💻 is connected to the wifi right now ? If yes or not try the process with the Ethernet cable connected directly from modem to your Mac and let me know how’s going
> > 
> > 
> > Seems you have very slow internet service usually takes few minutes downloading macOS Big Sur the installation takes about 25 to 30 minutes and few minutes optimizing macOS Big Sur
> 
> Just tried it, same error happens with Ethernet connection.

Ok let’s start over , what model is your Mac , graphics and HDD is installed ?and what you got installed additional with Catalina because I haven’t get any errors since  I did the upgrade  using patchet Sur I have help few guys with issues but you get this error

**Comment by bigmmind** [comment]
> > > > > > > > > USB 32gb , formatted in Mac OS Extended (Journaled) , GUID Partition
> > > > > > > > 
> > > > > > > > 
> > > > > > > > Good , download the patchet-Sur.dmg file but first turn off FileVault, then run patched Sur to create the bootable usb that isn’t create manually when all procedures finish reboot the Mac then turn it on holding down option key select EFI boot Mac will shut down then turn it on again then select install macOS Big Sur and let it finish
> > > > > > > 
> > > > > > > 
> > > > > > > Thank you, that is exactly how I did it, but still getting the error
> > > > > > 
> > > > > > 
> > > > > > Can take a picture with ur phone to look the error
> > > > > 
> > > > > 
> > > > > Here is a screenshot https://uploads-ssl.webflow.com/607d322572150dbaa2b1bac5/607e87ebb612d12a6e44f888_Screenshot%202021-04-20%20at%2010.49.44.png
> > > > 
> > > > 
> > > > Your Mac 💻 is connected to the wifi right now ? If yes or not try the process with the Ethernet cable connected directly from modem to your Mac and let me know how’s going
> > > 
> > > 
> > > Seems you have very slow internet service usually takes few minutes downloading macOS Big Sur the installation takes about 25 to 30 minutes and few minutes optimizing macOS Big Sur
> > 
> > 
> > Just tried it, same error happens with Ethernet connection.
> 
> Ok let’s start over , what model is your Mac , graphics and HDD is installed ?and what you got installed additional with Catalina because I haven’t get any errors since I did the upgrade using patchet Sur I have help few guys with issues but you get this error

Macbook Pro 15 Mid 2012
Graphics: Intel HD 4000 and Nvidia Gforce GT650M
12 GB RAM
Intel SSD 256GB (120GB Free space)
Mac OS Catalina 10.15.7 (19H524) , stock version nothing was tweaked.

**Comment by ozziepadilla** [comment]
Take a look here if you find your Mac model and if says it’s compatible with patchet Sur 
https://bensova.gitbook.io/big-sur/supported-macs

**Comment by bigmmind** [comment]
> Take a look here if you find your Mac model and if says it’s compatible with patchet Sur
> https://bensova.gitbook.io/big-sur/supported-macs

Yes, its says my Macbook is fully supported.

**Comment by ozziepadilla** [comment]
> > Take a look here if you find your Mac model and if says it’s compatible with patchet Sur
> > https://bensova.gitbook.io/big-sur/supported-macs
> 
> Yes, its says my Macbook is fully supported.

Good , are you in USA 🇺🇸?

**Comment by bigmmind** [comment]
No, I'm in Moscow, Russia :)

**Comment by ozziepadilla** [comment]
The usb drive you are using is brand new or you use before to storage files ? Did you have partitioned the HDD ? Or sdd?

**Comment by bigmmind** [comment]
Yes, I purchased a new usb stick and no I didnt partition my SSD

**Comment by ozziepadilla** [comment]
> Yes, I purchased a new usb stick and no I didnt partition my SSD

Ok let me talk with benSova I did all on my knowledge to help you don’t try to do anything for now I don’t know if after you tried to upgrade using Barry’s patchet cause the issues but I  am guessing let me ask benSova about it

**Comment by bigmmind** [comment]
Thanks a lot.

**Comment by bigmmind** [comment]
> > Yes, I purchased a new usb stick and no I didnt partition my SSD
> 
> Ok let me talk with benSova I did all on my knowledge to help you don’t try to do anything for now I don’t know if after you tried to upgrade using Barry’s patchet cause the issues but I am guessing let me ask benSova about it

I realized, when I load from the USB drive into the Big Sur recovery, there is no wifi icon in the corner, I tried to open a website in Safari in Recovery and it says no connection, even if I select my wifi network before loading from usb, so it appears that Big Sur doesn't see my internet at all, whether wifi or Ethernet.

**Comment by ozziepadilla** [comment]
> > > Yes, I purchased a new usb stick and no I didnt partition my SSD
> > 
> > 
> > Ok let me talk with benSova I did all on my knowledge to help you don’t try to do anything for now I don’t know if after you tried to upgrade using Barry’s patchet cause the issues but I am guessing let me ask benSova about it
> 
> I realized, when I load from the USB drive into the Big Sur recovery, there is no wifi icon in the corner, I tried to open a website in Safari in Recovery and it says no connection, even if I select my wifi network before loading from usb, so it appears that Big Sur doesn't see my internet at all, whether wifi or Ethernet.

This is your first time upgrading to macOS Big Sur ?

**Comment by BenSova** [comment]
> I realized, when I load from the USB drive into the Big Sur recovery, there is no wifi icon in the corner, I tried to open a website in Safari in Recovery and it says no connection, even if I select my wifi network before loading from usb, so it appears that Big Sur doesn't see my internet at all, whether wifi or Ethernet.

That's because Big Sur doesn't recognize older WiFi cards, that's why you have to patch the kexts after updating to macOS Big Sur.

Also, sorry, but I'm not 100% sure about what's going on with this problem, but I will look into it.

**Comment by ozziepadilla** [comment]
> > I realized, when I load from the USB drive into the Big Sur recovery, there is no wifi icon in the corner, I tried to open a website in Safari in Recovery and it says no connection, even if I select my wifi network before loading from usb, so it appears that Big Sur doesn't see my internet at all, whether wifi or Ethernet.
> 
> That's because Big Sur doesn't recognize older WiFi cards, that's why you have to patch the kexts after updating to macOS Big Sur.
> 
> Also, sorry, but I'm not 100% sure about what's going on with this problem, but I will look into it.

Hi Ben , I think he isn’t upgrading for first time to macOS Big Sur but seems that yes

**Comment by bigmmind** [comment]
This is my first time upgrading to Big Sur, I am currently on Catalina. Thanks both of you for all your efforts for looking into this!

**Comment by ozziepadilla** [comment]
> This is my first time upgrading to Big Sur, I am currently on Catalina. Thanks both of you for all your efforts for looking into this!

Ok , double check if your Mac with Catalina has the wifi on because should be on when you upgrade to Big Sur and creating the usb with patchet Sur , did you downloaded patchet-Sur.dmg file ?

**Comment by bigmmind** [comment]
> > This is my first time upgrading to Big Sur, I am currently on Catalina. Thanks both of you for all your efforts for looking into this!
> 
> Ok , double check if your Mac with Catalina has the wifi on because should be on when you upgrade to Big Sur and creating the usb with patchet Sur , did you downloaded patchet-Sur.dmg file ?

Yes, I have no issues with WiFi in Catalina and yes I downloaded the most recent release of patched-sur.dmg , only when I load from the USB , only then the wifi is not working.

**Comment by ozziepadilla** [comment]
> > > This is my first time upgrading to Big Sur, I am currently on Catalina. Thanks both of you for all your efforts for looking into this!
> > 
> > 
> > Ok , double check if your Mac with Catalina has the wifi on because should be on when you upgrade to Big Sur and creating the usb with patchet Sur , did you downloaded patchet-Sur.dmg file ?
> 
> Yes, I have no issues with WiFi in Catalina and yes I downloaded the most recent release of patched-sur.dmg , only when I load from the USB , only then the wifi is not working.

Ok , let’s do the easy way for me , do you use WhatsApp app ? If yes add me +14842725160 is much better for to talk because I am at work

**Comment by ozziepadilla** [comment]
Are you available to talk or FaceTime to guide you step by step ? If no it’s ok no problem

**Comment by bigmmind** [comment]
> Are you available to talk or FaceTime to guide you step by step ? If no it’s ok no problem

I'm jumping to work, but I could do tomorrow if you are still available. Thanks.

**Comment by ozziepadilla** [comment]
> > Are you available to talk or FaceTime to guide you step by step ? If no it’s ok no problem
> 
> I'm jumping to work, but I could do tomorrow if you are still available. Thanks.

Good so let me know because you have few hours over than me actually here thd tumdd es I’d 11:46 am New York time

**Comment by trombonebuster** [comment]
was this issue ever resolved? I'm having the same problem

**Comment by ozziepadilla** [comment]
> was this issue ever resolved? I'm having the same problem

Could you post a screenshot of the error please

**Comment by BenSova** [comment]
> was this issue ever resolved? I'm having the same problem

Did you try a clean install? Usually, this is a data incompatibility problem.

**Comment by trombonebuster** [comment]
Thanks for the quick response. I was getting errors in the installation log about not having an internet connection, so I plugged in my ethernet and now the error has changed to '...installer is damaged'. I am making a new installer now with patched sur and am downloading 11.4 now. Not sure if that will work. I tried a clean install once before using option R, but it crashed halfway and had clean install the original os

**Comment by trombonebuster** [comment]
If this doesn't work, I will try erasing from disk util in the installer and doing a clean install there

**Comment by ozziepadilla** [comment]
> Thanks for the quick response. I was getting errors in the installation log about not having an internet connection, so I plugged in my ethernet and now the error has changed to '...installer is damaged'. I am making a new installer now with patched sur and am downloading 11.4 now. Not sure if that will work. I tried a clean install once before using option R, but it crashed halfway and had clean install the original os

Ok I understand the error you are getting now “ the installer is damaged etc etc etc … right? “ then do this however if you do a clean install of upgrading then if you get the error saying the installer it’s damaged etc etc , hold down the power button until the Mac shuts off then turn it back on don’t press any key let it goes to the Apple logo then you will see the remaining time and the progress bar let it finish, if you think that is stucked don’t do anything let it complete the installation … when the installation is complete let it finish the optimizing then allow a least 5 to 10 minutes before Patchet Kexts and when it’s tikdd we to Patchet Kexts when it’s done reboot your Mac manually

**Comment by trombonebuster** [comment]
ok, thanks! I'll try it out.

**Comment by ozziepadilla** [comment]
> ok, thanks! I'll try it out.

It work you eill see,, this is the error you are getting right ? 
![BFCDF278-EF03-448E-8A0A-EB8C1F85EBAD](https://user-images.githubusercontent.com/78187370/120575414-0d241280-c3ef-11eb-99d0-e3f90605e43e.jpeg)

**Comment by trombonebuster** [comment]
yep thats it!

**Comment by ozziepadilla** [comment]
> yep thats it!

Ok go ahead leave the error don’t click on over the OK BUTTON

**Comment by ozziepadilla** [comment]
> yep thats it!

I am sorry for the typo I just edited to correcting … my eyes are tired

**Comment by trombonebuster** [comment]
haha no problem. I've been writing script all day and am the same. I actually didn't get an error this time, it rebooted to the progress bar/apple logo

**Comment by ozziepadilla** [comment]
> haha no problem. I've been writing script all day and am the same. I actually didn't get an error this time, it rebooted to the progress bar/apple logo

Great 👍 you got it some people get that error so Ben is aware of , but macOS Big Sur it’s installing anyway , good night

**Comment by trombonebuster** [comment]
Thanks so much for your help! It's been very annoying not being able to update Xcode in catalina

**Comment by ozziepadilla** [comment]
> Thanks so much for your help! It's been very annoying not being able to update Xcode in catalina

I know , by the way if the progress bar goes to 12 minutes remaining and takes long time like  in this screen and you see the loading wheel, do what I said before 
![9978BAFF-8947-446D-BF98-AED597B3412B](https://user-images.githubusercontent.com/78187370/120577170-f0d5a500-c3f1-11eb-81ef-310caef253dc.jpeg)

**Comment by trombonebuster** [comment]
k will do

**Comment by ozziepadilla** [comment]
> k will do

Good 👍

**Comment by trombonebuster** [comment]
Everything is installed and working great. Thanks again guys. I really can't afford a new mac right now.

**Comment by ozziepadilla** [comment]
> Everything is installed and working great. Thanks again guys. I really can't afford a new mac right now.

Glad it works for you

**Comment by craneo2099** [comment]
> Everything is installed and working great. Thanks again guys. I really can't afford a new mac right now.

Hi, Do you have any issues or problems developing or deploying to App Store with Xcode in not supported Mac?

I can't afford a new Mac too, but a i need Big Sur to work, Thanks in advance!

**Comment by sery321** [comment]
@ozziepadilla I have same problem with installation (installation cannot proceed because the installer is damaged). Could you help me via facetime? Tried multiple usb, redownloading Big Sur and solution via power button
